src/utils/protostar_directory.py

```diff
@@ -22,6 +22,13 @@ def protostar_binary_dir_path(self) -> Optional[Path]:
     def directory_root_path(self) -> Path:
         return self._protostar_binary_dir_path / ".." / ".."
 
+    def inject_protostar_cairo_dir(self, cairo_paths: List[Path]) -> List[Path]:
```

Comment:
this method is a bit weird. it pushes into the list and returns it as well. maybe a list spread would be more appropriate? and then we could name it `add_protostar_cairo_dir` instead of `inject`, the naming is a bit confusing for me as well

Reply:
`inject` is not the best choice. I'll change it to `add_protostar_cairo_dir_if_necessary`.